hello_nexus/basic/caller/app.py

```diff
@@ -0,0 +1,46 @@
+import asyncio
+import uuid
+from typing import Optional
+
+from temporalio.client import Client
+from temporalio.worker import UnsandboxedWorkflowRunner, Worker
+
+from hello_nexus.basic.caller.workflows import CallerWorkflow
+from hello_nexus.basic.service import MyOutput
+
+NAMESPACE = "my-caller-namespace"
+TASK_QUEUE = "my-caller-task-queue"
```

Comment:
Usually in samples, all task queues and workflow IDs are prefixed with something about the sample so they don't clash with other samples and are clear what they are for (and in this unique multi-namespace sample, I'd apply the same logic to namespace names and endpoint names too)